docs/frontend.md

```diff
@@ -0,0 +1,103 @@
+# Frontends
+
+A _frontend_ is a component of Chihaya that serves a BitTorrent tracker on one protocol.
+The frontend accepts, parses and sanitizes requests, passes them to the _backend_ (or _logic_) and writes responses to _Clients_.
```

Comment:
I thought about targeting both. The general public would be interested in the general stuff, like the functionality and the available frontends. Developers would, in addition to that, read the part about implementing a frontend.  Makes sense?